### PR TITLE
Simple UX for creating accounts

### PR DESCRIPTION
**Pull request by mgwalker**
Create a super simple UX for creating user accounts.

Addresses #1253 

- does not capture position or phone number, because the API doesn't support capturing those (which is weird, because those fields do exist in the database...)

### This pull request changes...
- adds a page at `/admin/create-account` for creating a new user account

   screen shot 2019-02-08 at 10 20 03 am

- pushing the yellow button will create an account and then blank out the form so you can create more accounts!

## Unhandled issues

- there's not a link to this page anywhere (see #1273)
- the system cannot send email, so someone will have to take responsibility for letting folks know their account was created and what their password is
- users cannot change their passwords yet (see #1274)
- the password strength meter isn't great; @jenniferthibault recommends segmenting it:
   4-choose-a-password
- leaving #1253 for now; this is just a first step, not the whole implementation

### This pull request is ready to merge when...
- [ ] Tests have been updated (and all tests are passing)
- [ ] This code has been reviewed by someone other than the original author
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented
- [ ] The change has been documented
  - [ ] Associated OpenAPI documentation has been updated

### This feature is done when...
- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

File: web/src/actions/admin.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  return axios
+    .post('/users', { email, name, password, state })
+    .then(() => {
+      dispatch({ type: ADMIN_CREATE_USER_SUCCESS });
+      dispatch(notify('User created!'));
+    })
+    .catch(e => {
+      dispatch({ type: ADMIN_CREATE_USER_ERROR });
+
+      switch (e.response.data.error) {
```
Comment (hbillings) [inline web/src/actions/admin.js]: This is one of the clearest patterns for handling this sort of interaction I've seen. :highfive:

File: web/src/containers/admin/CreateUser.js [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+      passwordQuality,
+      showPassword,
+      state
+    } = this.state;
+
+    // The password percent bar thing is 88% width, so scale
+    // the password score thing accordingly.  That's all.
+    const passwordPercent = 88 * passwordScore;
+
+    // Really the whole password meter should be replaced.
```
Comment (hbillings) [inline web/src/containers/admin/CreateUser.js]: We got an issue for that?

File: web/src/containers/admin/CreateUser.js [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+                onChange={this.handleChange}
+              />
+              <div className="right" style={{ width: '10%' }}>
+                {passwordQuality}
+              </div>
+              <div
+                className={`inline-block ${passwordColor} p1 rounded`}
+                style={{ width: `${passwordPercent}%` }}
+              />
+              <div className="alert-info p2">
```
Comment (hbillings) [inline web/src/containers/admin/CreateUser.js]: Can we make this tag a `p` without messing too much up?

File: web/src/containers/admin/CreateUser.js (hunk 2 above)
Comment (mgwalker) [reply web/src/containers/admin/CreateUser.js]: We do *not*, yes!  😬  Will add that issue.

File: web/src/containers/admin/CreateUser.js (hunk 3 above)
Comment (mgwalker) [reply web/src/containers/admin/CreateUser.js]: I reckon that's probably okay.  We'll see what happens...

File: web/package.json [hunk 4]
```diff
@@ -86,7 +86,8 @@
     "redux": "^3.7.2",
     "redux-thunk": "^2.3.0",
     "tippy.js": "^2.6.0",
-    "updeep": "^1.0.0"
+    "updeep": "^1.0.0",
+    "zxcvbn": "^4.4.2"
```
[the remark this answers is not in the document]
Comment (mgwalker) [reply web/package.json]: Yeah it's a really horrible name, but it's a good library.  Those goofs at Dropbox...

**Pull request merged**